addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/PassiveScanConfigJob.java

```diff
@@ -89,6 +73,34 @@ public void runJob(
         }
     }
 
+    @Override
+    public void runJob(
+            AutomationEnvironment env, LinkedHashMap<?, ?> jobData, AutomationProgress progress) {
+        // Configure any rules
+        Object o = jobData.get("rules");
+        ArrayList<?> ruleData = (ArrayList<?>) o;
+        for (Object rule : ruleData) {
+            if (rule instanceof LinkedHashMap<?, ?>) {
+                LinkedHashMap<?, ?> ruleMap = (LinkedHashMap<?, ?>) rule;
+                Integer id = (Integer) ruleMap.get("id");
+                PluginPassiveScanner plugin = getExtPScan().getPluginPassiveScanner(id);
+                AlertThreshold pluginTh =
+                        JobUtils.parseAlertThreshold(
+                                ruleMap.get("threshold"), this.getName(), progress);
+                if (pluginTh != null) {
+                    plugin.setAlertThreshold(pluginTh);
```

Comment:
Should test for `plugin != null` as well - in the verify we only `warn` if the plugin is null so users could have configured the env to ignore warnings.